client/my-sites/stats/features/modules/stats-devices/stats-module-upgrade-overlay.tsx

```diff
@@ -64,7 +64,7 @@ const StatsModuleUpgradeOverlay: React.FC< StatsModuleUpgradeOverlayProps > = (
 			splitHeader
 			overlay={
 				overlay ?? (
-					<StatsCardUpsellJetpack
+					<StatsCardUpsell
```

Comment:
StatsCardUpsell [determines whether to render the WPCOM or Jetpack upsell prompt](https://github.com/Automattic/wp-calypso/blob/9ffb111ae70470f6f0fbbb89aec8d218c99176fd/client/my-sites/stats/stats-card-upsell/index.tsx#L20), depending on whether the site is Jetpack (non-atomic) or not.